libs/pbd/fpu.cc

```diff
@@ -27,16 +27,18 @@
 #include <stdint.h>
 #include <assert.h>
 
-#ifdef PLATFORM_WINDOWS
-#include <intrin.h>
-#endif
-
 #ifdef ARM_NEON_SUPPORT
 /* Needed for ARM NEON detection */
 #include <sys/auxv.h>
 #include <asm/hwcap.h>
 #endif
 
+#ifdef PLATFORM_WINDOWS
+#include <intrin.h>
+#elif defined(__GNUC__) || defined(__clang__)
+#include <cpuid.h>
+#endif
```

Comment:
What happens if neither `PLATFORM_WINDOWS`  nor `__GNUC__` nor `__clang__` is defined?

Reply:
Previously that worked for other compilers (e.g. icc) with custom asm code.

Reply:
Changes are, there are no `cpuid.h` then. Should this be an `#error`?

Reply:
@pauldavisthefirst any opinion?  Should we just 
```
#error "Unsupported compiler"
```
or fall back to some asm code similar to the one already present (https://gcc.gnu.org/git/?p=gcc.git;a=blob;f=gcc/config/i386/cpuid.h#l221)